src/content/blog/v0.3_release.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "v0.3 release"
+pubDate: "2025-04-29"
+description: "Blog to accompany v0.3 release"
```

Comment:
I imagine once we update the index page to have a card for each blog post, the text in the description could display there? In that case we should edit this a little bit (e.g., something like "Updates on new features in v0.3 release"). Or we could just display the first sentence of the intro text there.